scripts/irx-soak.py

```
@@ -0,0 +1,251 @@
+
+
+def type_input(text):
+    if args.no_input:
+        return False
+    typed = subprocess.run(
+        ["idb", "ui", "text", "--udid", args.udid, text],
+        capture_output=True, text=True, timeout=30)
+    if typed.returncode != 0:
+        return False
+    # HID 40 = Return: submits the terminal composer.
+    subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)